drivers/flash/flash_stm32_ospi.c

```diff
@@ -1773,12 +1788,13 @@ static void flash_stm32_ospi_irq_config_func(const struct device *dev);
 
 PINCTRL_DT_DEFINE(STM32_OSPI_NODE);
 
+static const struct stm32_pclken pclken_id[] =
+		STM32_DT_INST_CLOCKS(0);
```

Comment:
This should fit in one line
```suggestion
static const struct stm32_pclken pclken_id[] = STM32_DT_INST_CLOCKS(0);
```